animata/widget/notes.tsx

```
@@ -1,6 +1,10 @@
 export function NotesCard({ title, children }: { title?: string; children?: React.ReactNode }) {
   return (
-    <div className="h-64 w-48 rounded-3xl border bg-[#fced99] p-4 font-sans text-zinc-950 shadow-sm">
+    <div
+      role="group"
+      tabIndex={0}
```

Biome (2.4.15)

[error] The HTML element div is non-interactive. Do not use tabIndex.

(lint/a11y/noNoninteractiveTabindex)